plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointSchemaManager.java

```diff
@@ -111,6 +112,13 @@ public CheckpointSchemaManager(TypeManager typeManager)
                 RowType.field("partitionValues", stringMap),
                 RowType.field("deletionTimestamp", BIGINT),
                 RowType.field("dataChange", BOOLEAN)));
+
+        sidecarEntryType = RowType.from(ImmutableList.<RowType.Field>builder()
```

Comment:
could this be a const?

Reply:
It uses `stringMap` that depends on injected TypeManager.